### ISSUE TITLE
Keda No Scale to zero when has multiple triggers 

### ISSUE THREAD

**Issue by joxpg** [kedacore/keda#4591]
### Report

Keda not scale to zero when combine trigger cron with cpu and memory utilization

### Expected Behavior

I hope keda scale to 0 when the deployment its outside pattern cron 

### Actual Behavior

Keda never scaledown to zero.

### Steps to Reproduce the Problem

1. I have this ScaleObject 
````
apiVersion: keda.sh/v1alpha1
kind: ScaledObject
metadata:
  name: api-qa-scaledobject
  namespace: api-qa
spec:
  maxReplicaCount: 10
  minReplicaCount: 0
  pollingInterval: 30
  cooldownPeriod: 30
  scaleTargetRef:
    apiVersion: apps/v1
    kind: Deployment
    name: api
  triggers: 
  - type: cron
    metadata: 
      timezone: America/Bogota
      start: 0 7 1-22 * 1-5
      end: 0 19 1-22 * 1-5
      desiredReplicas: "1"
  - type: cron
    metadata: 
      timezone: America/Bogota
      start: 0 7 23-31 * 1-5 
      end: 59 23 23-31 * 1-5 
      desiredReplicas: "1"
  - type: cpu
    metricType: Utilization # Allowed types are 'Utilization' or 'AverageValue'
    metadata:
      value: "50"
  - type: memory
    metricType: Utilization # Allowed types are 'Utilization' or 'AverageValue'
    metadata:
      value: "50"
````

### KEDA Version

2.10.1

### Kubernetes Version

1.25

### Platform

Amazon Web Services

**Comment by JorTurFer** [comment]
Hi
Your problem is related with using cpu/memory triggers, both scalers block the scaling to zero on any released version.
There is an already [merged PR removing this limitation](https://github.com/kedacore/keda/pull/4346) and allowing the scaling to zero if you use cpu/memory triggers but it'll be released as part of KEDA v2.11

I close this issue because it's already solved

**Comment by joxpg** [comment]
Thanks for the answer quickly, Maybe youu get date to release version 2.11? Regards
